### DISCUSSION TITLE
Player Count

### DISCUSSION THREAD

**Discussion by MrMoreira** [pelican-dev/panel#622] [Ideas]
Would you be able to add a native player count feature to the panel? Something like Minecraft’s display showing “5 players online” or “15/20 players,” but for other games as well. This would be really helpful for management. On the main dashboard, it could show something like “ARK: 50 players,” “Minecraft: 20 players,” and then a total player count across all servers, for example, "70 players in total."

**Comment by notAreYouScared** [comment]
This probably wouldn't be something we would add as a core feature of the panel. Could be a plugin when they come.

**Comment by Boy132** [comment]
Duplicate of https://github.com/pelican-dev/panel/discussions/123
